config/packages/kimai.yaml

```diff
@@ -138,19 +138,19 @@ kimai:
 #        day_limit: 4
 #        slot_duration: '00:30:00'
 #        businessHours:
-#            days: [1, 2, 3, 4, 5]
-#            begin: '08:00'
-#            end: '18:00'
+#           days: [1, 2, 3, 4, 5]
+#           begin: '07:00'
+#            end: '23:00'
 #        visibleHours:
 #            begin: '00:00'
 #            end: '24:00'
 #
 #        # You can configure unlimited google calendars to display events for your company (e.g. holidays)
 #        google:
-#            api_key: 'your-restricted-google-api-key'
+#            api_key: 'AIzaSyAgJlv3Z1ftGr5-ldvekBtRe8DXG5VtGGI'
```

Comment:
@kipester I think you've posted your private api key for your Google calender here! You should remove that and request a new api key from Google and disable the old one.

Reply:
OOps! Thank you!